internal/provider/apppackage.go

```diff
@@ -96,31 +96,16 @@ func resourceAppPackage() *schema.Resource {
 	}
 }
 
-// resourceAppPackageGetApp returns the given App from an appID. It returns an error if one was encountered.
-func resourceAppPackageGetApp(appID string, meta interface{}) (config.App, error) {
-	suite := meta.(config.Suite)
-
-	apps, err := suite.ExtrapolatedApps()
-	if err != nil {
-		return config.App{}, fmt.Errorf("resourceAppPackageGetApp unable to extrapolate apps: %s", err)
-	}
-
-	app, ok := apps.WithID(appID)
-	if !ok {
-		return config.App{}, fmt.Errorf("resourceAppPackageGetApp unable to find app with ID %q", appID)
-	}
-
-	return app, nil
-}
-
```

Comment:
This functionality was moved to the `config` abstraction package, so that it can be used with no code replication or dependency between resources or data sources.